controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php

```diff
@@ -114,31 +114,46 @@ function readInputData() {
 	}
 
 	/**
-	 * @copydoc Form::fetch()
+	 * @see Form::fetch()
```

Comment:
I think `@copydoc` was more appropriate here. Short story: when you want to pull in `@var` and `@return` from the parent self-documentation, use `@copydoc`. If you just want a soft link to the parent documentation, use `@see`.